common/lib/xmodule/xmodule/css/video/display.scss

```diff
@@ -21,7 +21,7 @@
 
 .video {
   @include clearfix();
-  background: rgb(240, 243, 245); // UXPL grayscale-cool xx-light;
+  background: rgb(245, 245, 245); // UXPL grayscale x-back
```

Comment:
I'll create a ticket to retroactively update this when that file is in. No point in making other variables we'll likely just forget about.